srtcore/sync.h

```diff
@@ -943,6 +943,42 @@ CUDTException& GetThreadLocalError();
 /// @param[in] maxVal maximum allowed value of the resulting random number.
 int genRandomInt(int minVal, int maxVal);
 
+
+// Implementation of a read-write mutex. 
+// This allows multiple readers at a time, or a single writer
```

Comment:
Doxygen-style comment.

```suggestion
/// Implementation of a read-write mutex. 
/// This allows multiple readers at a time or a single writer.
```